internal/db/schema/migrations/oss/postgres/100/04_resource.up.sql

```diff
@@ -0,0 +1,38 @@
+-- Copyright (c) HashiCorp, Inc.
+-- SPDX-License-Identifier: BUSL-1.1
+
+begin;
+
+  create table resource_enm (
+    string text not null primary key
```

Comment:
1. I think `string` is a poor column name since it is a primitive type in many languages and it doesn't convey what the values of the column represent. 
2. Shouldn't there be a constraint on this column since this is an enum table?